sparkler-api/src/main/java/edu/usc/irds/sparkler/util/FetcherDefault.java

```diff
@@ -39,6 +68,15 @@
     public FetchedData fetch(Resource resource) throws Exception {
         LOG.info("DEFAULT FETCHER {}", resource.getUrl());
         URLConnection urlConn = new URL(resource.getUrl()).openConnection();
+        if (httpHeaders != null){
+            httpHeaders.entrySet().forEach(e -> urlConn.setRequestProperty(e.getKey(), e.getValue()));
+        }
+        String userAgentValue = getUserAgent();
+        if (getUserAgent() != null) {
+            LOG.debug(USER_AGENT + ": " + userAgentValue);
+            urlConn.setRequestProperty(USER_AGENT, userAgentValue);
```

Comment:
That's right. I thought about it then came to simpler assumptions:
- If the user is not concerned about rotating agents (null or empty), then default will be picked
- If the user is concerned about the rotating agents, then we use the items explicitly specified in the list

I do not think we need to worry about default one because user has overridden it by using `fetcher.user.agents`.

what do you think? do you want me to append the default one into the rotating list?